shell/edit/namespace.vue

```diff
@@ -224,14 +230,24 @@ export default {
       <Tab
         v-if="!isView"
         name="labels-and-annotations"
-        label-key="generic.labelsAndAnnotations"
+        label-key="generic.labels"
+        :show-header="false"
         :weight="-1"
       >
-        <Labels
```

Comment:
Users should still be able to edit labels and annotations for a namespace

Reply:
Do we have 2 types of labels then? @kwwii could you please clarify this point?

Reply:
Yes, essentially this brings in a new type of label - a "system" label which can be shown or hidden by the user. A "system" label is a label that is used by Rancher itself to perform specific actions, in comparison to "user" labels which can be set by the user for use outside of the Rancher specific scope.

Reply:
What is the required action then? Should we keep `Label and Admission` or not?

Reply:
I assume you mean Annotations? If so, yes, there still needs to be a possibility for a user to enter a label or annotation that is not a PSA (or other possible system labels in the future).

Reply:
Ok but in these `Labels and Annotation` the user will see exactly the same `Labels` and on top of it the `Annotation`, so what's the sense of our custom design?

Reply:
The `Labels and Annotations` tab should still be shown. It's `Labels` component needs updating to have a button (situational given that it's only needed for PSA world?) to reveal the system labels (PSAs)

https://github.com/rancher/dashboard/issues/7119#issuecomment-1292511032

Reply:
Ok, i've seen it running again. The only thing visually missing here is the `Annotations`. The current `Labels` component which currently covers both labels and annotations should probably be beefed up to optionally include the `show system label` toggle.

Reply:
It's not the same, we are not using that logic somewhere else.

Reply:
After talking with @kwwii there are the changes to be applied:
- We can merge the 2 labels components
- Correct the Label title size and space to match the tab
- Border to be removed
- Toggler can be hidden if no system labels
- Description is kept

This new style is applied to all the `Labels and Annotations` tabs, which means that not having system labels, it will not have toggler and no disabled inputs.

Reply:
Updated and added screenshot.